src/library_html5.js

```diff
@@ -282,7 +282,11 @@ var LibraryHTML5 = {
       if (targetThread) JSEvents.queueEventHandlerOnThread_iiii(targetThread, callbackfunc, eventTypeId, keyEventData, userData);
       else
 #endif
-      if ({{{ makeDynCall('iiii', 'callbackfunc') }}}(eventTypeId, keyEventData, userData)) e.preventDefault();
+      if ({{{ makeDynCall('iiii', 'callbackfunc') }}}(eventTypeId, keyEventData, userData)) {
+        if(!(event.ctrlKey && event.key == 'v')) {
+          e.preventDefault();
```

Comment:
Perhaps it makes sense to special-case control-V here... I can see the motivation.

I worry that in general we don't have a principled approach to this in our JS libraries - I think SDL, glfw etc. each do their own thing. Which is not great.

It would be best to think about a general solution here. I'm not sure what that would be. But something that works for them all would be good. In theory, if we could have an API `emscripten_ignore_paste_commands()` that opts into this behavior, and does that for HTML5, SDL, glfw, etc., that seems best in theory, but I'm not sure if that's practical to do.

Reply:
Thank you @kripken for the comment. I'd assess my position here with this [meme](https://karms.biz/tmp/wasm.png).

So `emscripten_ignore_paste_commands()` would touch `library_html5.js`, `library_sdl.js`, `library_glfw.js`, `library_html5_webgl.js` and one could call it regardless of the "backend"?

Reply:
There may be a good argument here for an opt-in to this behaviour, or some other kind of switch.

@Karm, thanks for testing my little paste library - I'm just catching up with all the discussion, but it looks like the issue you've hit is SDL-specific.  I tested exclusively on GLFW, and had ctrl-V working consistently in that environment.

I would want to check that this proposed change doesn't break the existing behaviour for GLFW (as it already works as-is).  The easiest way to achieve that may indeed be just to have the behaviour made switchable.

Reply:
@slowriot I noticed that while Sokol based app, e.g. https://pthom.github.io/imgui_manual_online/manual_sokol/imgui_manual.html?version=1   works across the board even on tapping the screen in Firefox on my Android, your example (and my app) does not respond to such a tap attempt and thus doesn't let me paste on phone. 

So the events we are accidentally swallowing on SDL (and GLFW?) are also touch (tap?) and cmd on mac?

Reply:
I can't comment on SDL, but I don't believe GLFW is accidentally swallowing anything.  In my test application, I made no attempt to collect or process touch events.  To do so would require use of the [Emscripten touch api](https://emscripten.org/docs/api_reference/html5.h.html#touch) and then passing those inputs as mouse inputs to GLFW and hence to ImGui.  It could possibly benefit from some adaptation of [GLFW's multitouch feature](https://github.com/glfw/glfw/issues/42) as well.  Whereas with SDL, I believe all that comes out of the box, so there's no direct equivalence.  I think touch events are a red herring here.

Reply:
Ack. I don't have any meaningful analysis at hand. I merely gave it a try on my cellphone. Let's leave touch events out of this particular issue :+1: